### PR TITLE
 10171 add connector_dns module

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by elicoidal** [comment]
We should create the same module for dnspod.com (modifications should be 
light IMO)

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: connector_dns/tests/test_backend.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2015 Elico Corp (<http://www.elico-corp.com>)
```
Comment (siyuancn) [inline connector_dns/tests/test_backend.py]: no need full copyright here

**Comment by siyuancn** [comment]
Please user the structure:
A module is organised in a few directory:

```
controllers/: contains controllers (http routes)
data/: data xml
demo/: demo xml
models/: models definition
report/: reporting models (BI/analysis), Webkit/RML print report templates
static/: contains the web assets, separated into css/, js/, img/, lib/, ...
views/: contains the views and templates, and QWeb report print templates
wizards/: wizard model and views
```

File: connector_dns/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Credits
+=======
+
+
+Contributors
+------------
+
+* Liu Lixia <liu.lixia@elico-corp.com>
+* Augustin Cisterne-Kaas
```
Comment (elicoidal) [inline connector_dns/README.rst]: augustin.cisternekaas@elico-corp.com

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: connector_dns/README.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+==============
+Connector Dns
```
Comment (elicoidal) [inline connector_dns/README.rst]: DNS

**Comment by elicoidal** [comment]
LGTM

**Comment by Rona111** [comment]
@elicoidal  need remove author OCA?

**Comment by elicoidal** [comment]
indeed

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: connector_dns/README.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ * have basic modules installed (connector)
+
+
+Usage
+=====
+
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
```
Comment (siyuancn) [inline connector_dns/README.rst]: I don't think https://www.odoo.com/forum/help-1 is helpful if user has any question about this module.
Shall we put our link here?

Comment (Rona111) [inline connector_dns/README.rst]: @siyuancn I agree with you. so,should i remove the link? @elicoidal

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by noahzaozao** [comment]
@elicoidal @seb-elico For connector_dns, since the PR OCA/l10n-china are not able to merge , https://github.com/OCA/infrastructure-dns/pull/4 , shall we merge this PR? I'll create new PR with my changes after this one.

**Comment by elicoidal** [comment]
I need to adapt the README in your repo and I will push to merge.

**Comment by elicoidal** [comment]
I mean merge in OCA @noahzaozao

**Pull request closed without merging**